script/benchmark.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const { performance, PerformanceObserver } = require('perf_hooks');
+const { spawnSync, spawn } = require('child_process');
```

Comment:
hexo-util has [`spawn`](https://github.com/hexojs/hexo-util#spawncommand-args-options) method

Reply:
I see that, but it's returns a promise which I couldn't add listener on it